configure.ac

```diff
@@ -49,6 +49,7 @@ have_fuse_opt_parse=no
 oldlibs="$LIBS"
 LIBS="$LIBS $SSHFS_LIBS"
 AC_CHECK_FUNC([fuse_opt_parse], [have_fuse_opt_parse=yes])
+AC_CHECK_FUNC([fsp_fuse_opt_parse], [have_fuse_opt_parse=yes])
```

Comment:
The original version of the WinFsp FUSE compatibility layer needed this. This is because the original compatibility layer exposed a function `fsp_fuse_opt_parse` and the `fuse_opt_parse` symbol was really a macro.

Things have changed since and the `AC_CHECK_FUNC` line is no longer needed with the CYGFUSE project: https://github.com/mgeisert/cygfuse

Please allow me a few hours to double-check that everything still works for you.